kermit-io/build.gradle.kts

```diff
@@ -62,7 +62,7 @@ kotlin {
         commonMain.dependencies {
             implementation(project(":kermit-core"))
 
-            api(libs.kotlinx.datetime)
+            implementation(libs.kotlinx.datetime)
```

Comment:
The reason I set it up as an api dependency is because `kotlinx.datetime.Clock` is exposed in the constructor.

Reply:
Oh I didn't do that... ok. I should have! I think this should still be an API dependency and we let Gradle do version conflict resolution.

Reply:
Ok, we'll keep it as `api` for now.